ChangeLog.md

```diff
@@ -1,5 +1,13 @@
 # Sourdough
 
+## 0.10.0
+
+Deal with bad behavior by VSphere/VMWare. We sometimes see our VSphere host reject connections, causing some chef runs to get the default runlist instead of the real one.
+
+To deal with this, we write the tag data to a knob file, so on future runs we load from there and don't need to care if VSphere is having a tizzy.
+
+This does mean that changing a VSphere tag will not affect the parameters of a given VM, but in our use case, we never change the tags after a VM is spun up anyway.
+
```

Comment:
I suggested in the ticket to re-order the process to avoid this limitation...

> Desired Behavior: I suggest we reorder this because vcenter should take priority over everything else and if vcenter is unreachable it should look at a local cache (ie. /etc/knobs/Runlist). The order I suggest is look at vcenter first, if there is a runlist then save it in /etc/knobs/Runlist, only if vcenter is unreachable or the runlist missing then use the /etc/knobs/Runlist. If nothing else exists then default to 'recipe[clh_base_server]'.

Is there a reason you could not do it this way?